### PR TITLE
updated T and asd from O2 to O1

### PR DESCRIPTION
**Pull request by nihargupte-ph**

### CODE REVIEW

File: examples/toy_npe_model/asd_dataset_settings.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ dataset_settings:
 #  f_max: 2048      # defaults to f_s/2
   f_s: 4096
   time_psd: 1024
-  T: 8
+  T: 4
```
Comment (stephengreen) [inline examples/toy_npe_model/asd_dataset_settings.yaml:6]: We're using 4 s PSDs here; is that consistent with the rest of the "toy example" settings?

File: examples/toy_npe_model/training/local_settings.yaml
Comment (stephengreen) [inline examples/toy_npe_model/training/local_settings.yaml:1]: Don't save the local settings.

File: examples/toy_npe_model/training/model_latest.pt
Comment (stephengreen) [inline examples/toy_npe_model/training/model_latest.pt:1]: Don't save networks.

File: examples/toy_npe_model/training_data/asd_dataset/tmp/O1/psd_time_segments.pkl
Comment (stephengreen) [inline examples/toy_npe_model/training_data/asd_dataset/tmp/O1/psd_time_segments.pkl:1]: Remove data files.

**Review by stephengreen — CHANGES_REQUESTED** [review]
Thanks. I think you accidentally committed data files. I also want to confirm that 4s is being used throughout the toy example.

File: examples/toy_npe_model/asd_dataset_settings.yaml (hunk 1 above)
Comment (nihargupte-ph) [reply examples/toy_npe_model/asd_dataset_settings.yaml]: Right, it was changed to 8 in the last commit which broke the example. So I changed it back to 4 to get things to work.

File: examples/toy_npe_model/training/model_latest.pt
Comment (nihargupte-ph) [reply examples/toy_npe_model/training/model_latest.pt:1]: oops, I rewrote the history to fix this....

**Review by stephengreen — CHANGES_REQUESTED** [review]
I think the train settings file (and associated docs) for the toy model also has to be updated to 4s.

**Review by stephengreen — APPROVED** [review]

**Pull request merged**